src/ui/List/DescriptionList.stories.js

```diff
@@ -48,42 +41,37 @@ export default {
   },
 };
 
-function DescriptionList(items, floated, animated, divided, horizontal) {
+function DescriptionList(items, floated, animated, divided) {
   return (
-    <List animated={animated} divided={divided} horizontal={horizontal}>
-      {items.map((item, index) => (
-        <List.Item key={index}>
-          <List.Content>
-            <List.Header>{item.header}</List.Header>
-            <List.Description floated={floated}>
-              {item.content}
-            </List.Description>
-          </List.Content>
-        </List.Item>
-      ))}
-    </List>
+    <Container>
+      <List animated={animated} divided={divided}>
```

Comment:
@g-stamatis All list should use HTML semantic elements for listing. Thus, here we should use:

```
<dl>
  <dt></dt>
   <dd></dd>
</dl>
```